newsroom/news_api/default_settings.py

```diff
@@ -33,7 +33,6 @@
     "newsroom.products",
     "newsroom.news_api.api_audit",
     "newsroom.news_api.news.assets.assets",
-    "newsroom.upload",
```

Comment:
Would we need `MODULES = ["newsroom.assets.module"]` here so image downloads from the NewsAPI work?